cmd/kubeadm/app/discovery/token/token.go

```diff
@@ -205,8 +210,21 @@ func runForEndpointsAndReturnFirst(endpoints []string, fetchKubeConfigFunc func(
 			}, constants.DiscoveryRetryInterval, stopChan)
 		}(endpoint)
 	}
-	wg.Wait()
-	return resultingKubeConfig
+	timeout := constants.DefaultDiscoveryTimeout
```

Comment:
Set the default value for field `DiscoveryTimeout` in `SetDefaults_NodeConfiguration` instead of defining here?